src/data/triggers.json

```diff
@@ -66,6 +66,7 @@
   "juice is worth the squeeze",
   "key takeaway(|s)",
   "knowledge champion(|s)",
+  "ladder(|s) up"
```

Comment:
@gmkohler could you add a trailing comma here? ie `"ladder(|s) up",`

Reply:
@okize done.

Separately, I'm wondering whether this syntax would allow for `ladder`,`ladders`,and `laddering`?:

`ladder(|s|ing)`

Reply:
@gmkohler yep! I believe `ladder(|s|ing) up` would trigger on `ladder up`, `ladders up` and `laddering up`.